pkgs/development/libraries/fastpbkdf2/default.nix

```diff
@@ -0,0 +1,28 @@
+{ stdenv, fetchurl, openssl }:
+
+stdenv.mkDerivation {
+  name = "fastpbkdf2-1.0.0";
+  
+  src = fetchurl {
+    url = https://github.com/ctz/fastpbkdf2/archive/v1.0.0.tar.gz;
+    sha256 = "492874e2088dd38cfa74b1b3a99c0b907ecb8da96c656e618353a2e825b8c152";
+  };
+  
+  buildInputs = [ openssl ];
+
+  preBuild = ''
+    makeFlagsArray=(CFLAGS="-std=c99 -O3 -g")
```

Comment:
`-O3` doesn't seem to be particularly safe for general use.

Reply:
There are many packages that have `-O3` in their `CFLAGS`, but I agree that it may not be portable across different platforms. Do you know of a more portable way to enable optimization in the build?

Reply:
I couldn't find a better mechanism in Nixpkgs, so nevermind. I thought `-O2` might be better, but on a general note it would be better if optimization flags were set somewhere else and not in the derivation.